usr/src/uts/common/brand/lx/procfs/lx_prvnops.c

```diff
@@ -5156,6 +5233,20 @@ lxpr_read_sys_kernel_threads_max(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
 	lxpr_uiobuf_printf(uiobuf, "%d\n", LXPTOZ(lxpnp)->zone_nlwps_ctl);
 }
 
+/*
+ * Some applications check /proc/sys/kernel/panic_on_oops, the purpose
+ * of setting this bit in Linux is to call panic() in case of
+ * an 'oops' (a serious non-fatal error in the kernel).
+ * So we just returned 1 as is the expected value for some applications.
+ */
+static void
+lxpr_read_sys_kernel_panic_on_oops(lxpr_node_t *lxpnp, lxpr_uiobuf_t *uiobuf)
+{
+	uint_t val = 1;
+	ASSERT(lxpnp->lxpr_type == LXPR_SYS_KERNEL_PANIC_ON_OOPS);
+	lxpr_uiobuf_printf(uiobuf, "%u\n", val);
+}
```

Comment:
One COULD, if they tilt their head hard enough, check for a DEBUG kernel here.  Not sure if that's in-scope for this "just lie about it for now".  Also, is it normally set to 1?

Reply:
@danmcd According to this documentation, I don't see a default value for this option. https://www.kernel.org/doc/html/latest/admin-guide/sysctl/kernel.html#panic-on-oops
For example in https://docs.oracle.com/en/database/oracle/oracle-database/19/ladbi/minimum-parameter-settings-for-installation.html#GUID-CDEB89D1-4D48-41D9-9AC2-6AD9B0E944E3 requires that  /proc/sys/kernel/panic_on_oops should be set to 1, so I think little harm is done to put this to that expected value. I checked Oracle Linux and by default this sysctl is set to 1. 
If needed we make it an RW tunable but values should be  constrained to 0-1.